src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -529,9 +529,7 @@ public boolean verifyEndDatesBeforeTodayAreDisabled() {
             String year = Integer.toString(cal.get(Calendar.YEAR));
 
             // go to previous month
-            while (!(getDatepickerMonth().equals(month) && getDatepickerYear().equals(year))) {
-                browser.driver.findElement(By.className("ui-datepicker-prev")).click();
-            }
+            navigateToRightMonthAndYear(month, year);
```

Comment:
How do we ensure that the javascript executor has finished running before proceeding?